package.json

```diff
@@ -114,6 +115,6 @@
     "@std-proposal/temporal": "0.0.1"
   },
   "optionalDependencies": {
-    "@jobber/fonts": "^1.0.0"
+    "@jobber/fonts": "^1.1.0"
```

Comment:
Oh nice! While we're at it. Mind updating this require

https://github.com/GetJobber/atlantis/blob/352545921cad94f2129c55389c3880bc7f57ea2e/gatsby-browser.js#L2-L3

to

```ts
  require("@jobber/fonts")
```

without the disable

Reply:
@darryltec done